openid-sse-framework-1_0.xml

```diff
@@ -887,7 +775,342 @@ Cache-Control: no-store
+  },
+  {
+    "stream_id": "50b2d39934264897902c0581ba7c21a3",
+    "iss": "https://tr.example.com",
+    "aud": [
+        "http://receiver.example.com/web",
+        "http://receiver.example.com/mobile"
+      ],
+    "delivery": {
+      "delivery_method":
+        "https://schemas.openid.net/secevent/risc/delivery-method/push",
+      "url": "https://receiver.example.com/events"
+    },
+    "events_supported": [
+      "urn:example:secevent:events:type_1",
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3"
+    ],
+    "events_requested": [
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3",
+      "urn:example:secevent:events:type_4"
+    ],
+    "events_delivered": [
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3"
+    ]
+  }
+]
+]]></artwork>
+            </figure>
+
+
+            <t>The following is a non-normative example response to a request with no "stream_id"
+            when there is only one Event Stream configured:</t>
+
+            <figure title="Example: Read Stream Configuration Response" anchor="figreadconfigrespnostreamidonestream">
+              <artwork><![CDATA[
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+[
+  {
+    "stream_id": "f67e39a0a4d34d56b3aa1bc4cff0069f",
+    "iss": "https://tr.example.com",
+    "aud": [
+        "http://receiver.example.com/web",
+        "http://receiver.example.com/mobile"
+      ],
+    "delivery": {
+      "delivery_method":
+        "https://schemas.openid.net/secevent/risc/delivery-method/push",
+      "url": "https://receiver.example.com/events"
+    },
+    "events_supported": [
+      "urn:example:secevent:events:type_1",
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3"
+    ],
+    "events_requested": [
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3",
+      "urn:example:secevent:events:type_4"
+    ],
+    "events_delivered": [
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3"
+    ]
+  }
+]
+]]></artwork>
+            </figure>
+
+            <t>The following is a non-normative example response to a request with no "stream_id"
+            when there are no Event Streams configured:</t>
+
+            <figure title="Example: Read Stream Configuration Response" anchor="figreadconfigrespnostreamidnostreams">
+              <artwork><![CDATA[
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+[]
+]]></artwork>
+            </figure>
+
+            <t>Errors are signaled with HTTP status codes as follows:</t>
+
+            <texttable title="Read Stream Configuration Errors" anchor="tabreadconfig">
+              <ttcol align="left">Code</ttcol>
+              <ttcol align="left">Description</ttcol>
+              <c>401</c>
+              <c>if authorization failed or it is missing</c>
```

Comment:
"if credentials are missing or invalid"

Reply:
I'd prefer to leave these consistent with the other error definitions in the spec. We can create a new issue to discuss changing all of them at once.